server/static/style.css

```diff
@@ -1,5 +1,62 @@
 /* Downloaded from: https://github.com/23/resumable.js/blob/master/samples/Node.js/public/style.css */
 
+/* Move down content because we have a fixed navbar that is 50px tall */
+body {
+  padding-top: 50px;
+}
+
+/*
+ * Global add-ons
+ */
+
+.sub-header {
+  padding-bottom: 10px;
+  border-bottom: 1px solid #eee;
+}
+
+/*
+ * Top navigation
+ * Hide default border to remove 1px line.
+ */
+.navbar-title {
+    font-size: 18px;
+    color: white;
+    padding: 15px;
+    cursor: default;
+}
+
+.navbar-fixed-top {
+  border: 0;
+}
+
+.tab-label,
+.tab-label:hover {
+    font-size: 14px;
+    color: black;
+}
+
+.main,
+.sample-table {
+    margin-top: 30px;
+}
+
+.step-header {
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+.step-description {
+    text-align: justify;
+    margin-bottom: 10px;
+}
+
+@media (min-width: 768px) {
+  .footer-row {
+    padding-right: 40px;
+    padding-left: 40px;
+  }
```

Comment:
You can probably do this with bootstrap col classes instead of your own custom media query.